tamr_client/datasets/dataframe.py

```diff
@@ -0,0 +1,38 @@
+"""
+Convenient functionality for interacting with pandas DataFrames.
+"""
+
+import json
+
+import pandas as pd
+
+import tamr_client as tc
+from tamr_client.datasets.dataset import Dataset
```

Comment:
You can just access `Dataset` via `tc.Dataset`, so you can remove this import. Same could go for `tc.record.PrimaryKeyNotFound`. I think its good practice to use our library internally the same way users would use it, if possible.

Note: I treat `tamr_client.types` as a special case because these emulate types that should exist in the standard library, so I don't like accessing them via `tc.`

Reply:
Done.